libexec/master-agitator.pl

```diff
@@ -30,6 +30,12 @@
   print "ERROR: ACCUMULO_HOME needs to be set!";
   exit(1);
 }
+if( defined $ENV{'PSSH'} ){
+  $PSSH = $ENV{'PSSH'};
+} else {
+  print "ERROR: PSSH needs to be set!";
+  exit(1);
+}
```

Comment:
I am not sure why but this error is being printed for me. Both commands `pssh` and `parallel-ssh` work on my machine.

Reply:
Running `export PSSH=parallel-ssh` will solve this but I thought the changes in `start_agitator()` in the agitator script were supposed to automatically set the env variable.

Reply:
Oops, the variable needs to be exported to be seen by the perl script. Should be fixed in 6a665e7a1c995c0f7f40aa1dfde29d515ed706fe